src/main/scala/org/broadinstitute/dsde/workbench/sam/config/AppConfig.scala

```diff
@@ -153,14 +153,23 @@ object AppConfig {
     )
   }
 
-  implicit val azureServicesConfigReader: ValueReader[AzureServicesConfig] = ValueReader.relative { config =>
-    AzureServicesConfig(
-      config.as[Option[Boolean]]("azureEnabled"),
-      config.getString("managedAppClientId"),
-      config.getString("managedAppClientSecret"),
-      config.getString("managedAppTenantId"),
-      config.as[Seq[String]]("managedAppPlanIds")
-    )
+  implicit val azureServicesConfigReader: ValueReader[Option[AzureServicesConfig]] = ValueReader.relative { config =>
+    config
+      .getAs[Boolean]("azureEnabled")
+      .flatMap(azureEnabled =>
+        if (azureEnabled) {
```

Comment:
Since it wasn't being read before, is `azureEnabled` set to true in dev? I don't see it here: https://github.com/broadinstitute/firecloud-develop/blob/dev/base-configs/sam/sam.conf.ctmpl#L284

(Want to make sure we don't turn off azure in dev)